src/blockchain.py

```diff
@@ -570,13 +582,76 @@ def find_fork_point(self, alternate_chain: List[bytes32]) -> uint32:
         # TODO: 17. check cost
         return True
 
-    async def validate_block(self, block: FullBlock, genesis: bool = False) -> bool:
+    async def pre_validate_blocks(self, blocks: [FullBlock]) -> ([bool, bytes32]):
+        data = []
+        #cbor used because of multiprocessing auto pickle fails
```

Comment:
You can use bytes(block), which uses Streamable class. (and then use FullBlock.from_bytes(,...))

Reply:
thx, that's much cleaner